src/java/picard/vcf/LiftoverVcf.java

```diff
@@ -281,6 +311,7 @@ public static void main(final String[] args) {
                 else {
                     sorter.add(builder.make());
                 }
+                reverseComplementAlleleMap.clear();
```

Comment:
@tfenne I am wondering if this line may be fixing my heap problems instead of the new reference management code?